src/main/java/net/dv8tion/jda/api/requests/Route.java

```diff
@@ -433,10 +387,7 @@ public boolean equals(Object o)
     @Override
     public String toString()
     {
-        return new EntityString(this)
-                .setType(method)
-                .addMetadata("route", route)
-                .toString();
+        return method + "/" + route;
```

Comment:
Looks like a regression. Likely from before the EntityString PR as merged.

Reply:
This change is intentional, the toString is used for logging and I'd like to keep that one simple.

Reply:
`Route[Put](route=applications/{application_id}/commands)`
`Put/applications/{application_id}/commands`

Between these two, i definitely think the top one is better.